src/web_app_skeleton/config/database.cr.ecr

```diff
@@ -7,7 +7,9 @@ Avram::Repo.configure do |settings|
     settings.url = ENV["DATABASE_URL"]? || Avram::PostgresURL.build(
       database: database_name,
       hostname: ENV["DB_HOST"]? || "localhost",
+      # Common default usernames may be "postgres", "root", or your system username
       username: ENV["DB_USERNAME"]? || "postgres",
+      # If no password is necessary, use an empty string ""
```

Comment:
Might be worth saying that sometimes default Postgres credentials require no password?

```
# Some Postgres installations require no password. Use "" if that is the case.
```

Thoughts on that? Does it make sense